app/scripts/modules/amazon/webpack.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
       mangle: false,
       beautify: true,
       comments: false,
-      sourceMap: true,
+      sourceMap: false,
```

Comment:
this broke again (it happens) - will try to dig into it this afternoon and get it fixed.